bin/get-pact

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -e
 
-curl -LO https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v1.64.0/pact-1.64.0-linux-x86_64.tar.gz
-tar xzf pact-1.64.0-linux-x86_64.tar.gz
-sha256sum -c "$CI_DEPLOY_DIR/data/pact-sha256sum" < pact-1.64.0-linux-x86_64.tar.gz
+if [ -d "$CI_DEPLOY_DIR/pact" ]; then
+  echo "Pact already downloaded... Skipping"
```

Comment:
This doesn't feel right, we should definitely at least still be checking the sha256 of the file instead of blindly trusting